docs/source/etcd-data-model.rst

```diff
@@ -265,6 +265,22 @@ The various properties in this object have the following meanings:
 
 .. _security-profile-data:
 
+Traffic Filtering
+~~~~~~~~~~~~~~~~~
+
+Calico implements traffic filtering controlled by security profiles
+and tiered security policies.  Traffic is connection-like: the
```

Comment:
How about making this `connection-based` instead of `connection-like`?  I think that'd read better.